### DISCUSSION TITLE
Hyperformula support for ag grid

### DISCUSSION THREAD

**Discussion by veenava** [handsontable/hyperformula#817] [General]
Unable to find any documentation for hyperformula support for Ag grid. Has anyone implemented the same? Any help or guidance would be appreciated.

**Comment by AMBudnik** [comment]
Hi @veenava 

The only supported integration of HyperFormula with a spreadsheet solution with an advanced scope of features is with the Handsontable. You can read more about it here https://handsontable.com/docs/formula-calculation/#passing-the-hyperformula-class-instance-to-handsontable. 

If you would have more questions, please send an email to support@handsontable.com
